src/common/camera_control.c

```diff
@@ -444,8 +476,14 @@ static void _camera_process_job(const dt_camctl_t *c, const dt_camera_t *camera,
       if(gp_widget_get_child_by_name(config, spj->name, &widget) == GP_OK)
       {
         int value = spj->value;
-        gp_widget_set_value(widget, &value);
-        gp_camera_set_config(cam->gpcam, config, c->gpcontext);
+        int set_value_succeeds = gp_widget_set_value(widget, &value);
+        if(set_value_succeeds != GP_OK){
```

Comment:
dt style is to have `{` on next line. There is multiple instance of this below (also after `else`).

Reply:
fixed